config/locales/en.yml

```
@@ -100,7 +100,10 @@ en:
         unauthorized_client: 'The client is not authorized to perform this request using this method.'
         access_denied: 'The resource owner or authorization server denied the request.'
         invalid_scope: 'The requested scope is invalid, unknown, or malformed.'
-        invalid_code_challenge_method: 'The code challenge method must be plain or S256.'
+        invalid_code_challenge_method:
+          zero: 'There are no acceptable code_challenge methods'
```

this should never be reachable, but if it is, say
```suggestion
          zero: 'This authorization server does not support PKCE as there a no accepted code_challenge methods'
```

Would be better?

---

That is better, tweaked it just a little bit to align with other messages and updated in [f042917](https://github.com/doorkeeper-gem/doorkeeper/pull/1747/commits/f04291758eae274b9fc92a141e71b7d4aeae8c46)